remote-api/src/main/java/org/alfresco/rest/api/impl/NodesImpl.java

```diff
@@ -241,9 +243,12 @@
 
     private Set<String> nonAttachContentTypes = Collections.emptySet(); // pre-configured whitelist, eg. images & pdf
 
-    public void setNonAttachContentTypes(Set<String> nonAttachWhiteList)
+    public void setNonAttachContentTypes(String nonAttachAllowListStr)
     {
-        this.nonAttachContentTypes = nonAttachWhiteList;
+        if (nonAttachAllowListStr != null)
+        {
+            nonAttachContentTypes = Stream.of(nonAttachAllowListStr.trim().split("\\s*,\\s*")).collect(Collectors.toSet());
```

Comment:
Wouldn't following be a bit simpler?
```suggestion
            nonAttachContentTypes = Set.of(nonAttachAllowListStr.split(","));
```

Reply:
Thanks - done (with tweaks) - see also earlier comment.